draft-ietf-jsonpath-base.md

```diff
@@ -449,13 +449,15 @@ return an error.
 
 ## Syntax
 
-Syntactically, a JSONPath consists of a root selector (`$`), which
-selects the root node of a JSON value, followed by a possibly empty
-sequence of *selectors*.
+Syntactically, a JSONPath consists of either a root selector (`$`), which
+selects the root node of a JSON value, or a current item selector (`@`),
```

Comment:
This doesn't say what a current item selector actually selects.
(The other text also needs to be fixed, for which I'll probably generate a separate PR.)

Reply:
Thanks for your early feedback, remeber that it is still a draft / WIP.
It is not complete and it might contain typos and other mistakes.

Reply:
@cabo let me know if the changes to this paragraph are now ok for you. Let me know if you have any further advice, otherwise I feel like we can resolve this conversation.

Reply:
@cabo: just pushed further changes to this, I think that this conversation is outdated and it can be archived. Let me know if you have any further feedback.

Reply:
Sorry, github does not make it easy to find the current state of a PR where a conversation has led to changes.  So please ignore my comments if they relate to outdated contents.

I think the specific wordsmithing of this PR can wait until some other parts of the draft get more stable.